contrib/win32/win32compat/ansiprsr.c

```diff
@@ -176,14 +178,10 @@ unsigned char* ParseBuffer(unsigned char* pszBuffer, unsigned char* pszBufferEnd
 			unsigned char * pszCurrent = pszBuffer+1;
 			unsigned char * pszNewCurrent = pszCurrent;
 
-			if (term_mode == TERM_ANSI)
+			if (term_mode == TERM_ANSI && bAnsiParsing)
```

Comment:
The first condition "term_mode == TERM_ANSI" will always be true as it is static at line 72. what is the need for having it standard? why can't we read the term setting of the console?